maintenance/wikia/PortabilityMetric/updateMainPageFilterInMetricProcessor.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @ingroup Maintenance
+ *
+ * @desc For current wiki, find an ID of it's Main Page
```

Comment:
offline discussion: how should we tackle a possible change of the wiki Main Page?
Some possible options:
- clean a table row before each run of the script
- let script to update *all* pages on the wiki each time